iphone/Classes/TiBluetoothPeripheralManagerProxy.m

```diff
@@ -67,7 +67,25 @@ - (void)startAdvertising:(id)args
     args = [args objectAtIndex:0];
     
     if ([args isKindOfClass:[NSDictionary class]]) {
-        [peripheralManager startAdvertising:args];
+        NSMutableDictionary *advertisementData = [NSMutableDictionary dictionary];
+        NSString *localNameKey;
+        NSString *serviceUUIDsKey;
+        
+        ENSURE_ARG_OR_NIL_FOR_KEY(localNameKey, args, @"localNameKey", NSString);
+        ENSURE_ARG_OR_NIL_FOR_KEY(serviceUUIDsKey, args, @"serviceUUIDsKey", NSArray);
+        
+        if (localNameKey) {
+            [advertisementData setObject:localNameKey
+                        forKey:CBAdvertisementDataLocalNameKey];
```

Comment:
I see, then let's keep it flexible. I did so because according to [https://developer.apple.com/documentation/corebluetooth/cbperipheralmanager/1393252-startadvertising](url), only those 2 keys are supported